crates/matrix-sdk-crypto/src/identities/manager.rs

```diff
@@ -756,105 +756,111 @@ pub(crate) mod testing {
             .expect("Can't parse the keys upload response")
     }
 
-    pub fn own_key_query() -> KeyQueryResponse {
-        let data = response_from_file(&json!({
-          "device_keys": {
-            "@example:localhost": {
-              "WSKKLTJZCL": {
-                "algorithms": [
-                  "m.olm.v1.curve25519-aes-sha2",
-                  "m.megolm.v1.aes-sha2"
-                ],
-                "device_id": "WSKKLTJZCL",
-                "keys": {
-                  "curve25519:WSKKLTJZCL": "wnip2tbJBJxrFayC88NNJpm61TeSNgYcqBH4T9yEDhU",
-                  "ed25519:WSKKLTJZCL": "lQ+eshkhgKoo+qp9Qgnj3OX5PBoWMU5M9zbuEevwYqE"
-                },
-                "signatures": {
-                  "@example:localhost": {
-                    "ed25519:WSKKLTJZCL": "SKpIUnq7QK0xleav0PrIQyKjVm+TgZr7Yi8cKjLeZDtkgyToE2d4/e3Aj79dqOlLB92jFVE4d1cM/Ry04wFwCA",
-                    "ed25519:0C8lCBxrvrv/O7BQfsKnkYogHZX3zAgw3RfJuyiq210": "9UGu1iC5YhFCdELGfB29YaV+QE0t/X5UDSsPf4QcdZyXIwyp9zBbHX2lh9vWudNQ+akZpaq7ZRaaM+4TCnw/Ag"
+    pub fn own_key_query_with_user_id(user_id: &UserId) -> KeyQueryResponse {
+        {
```

Comment:
Extra block scope covering the entire function?

Reply:
Removed in https://github.com/matrix-org/matrix-rust-sdk/pull/1427/commits/1a7c5920622d686cc618f1f8dd8a2bd11c2c7b59

Reply:
Oops, thanks for spotting. poljar's commit fixed it.